### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by hovkaren** [tranngocduvnvp/Img2LaTex#1]
Is it possible provide documentation to how install this package and use in python server.

**Comment by tranngocduvnvp** [comment]
Hi
Documentation for the installation has been updated in the install section of the repo.

**Comment by hovkaren** [comment]
Thanks @tranngocduvnvp

**Comment by hovkaren** [comment]
I have done installation steps. Now I have `img2tex` package folder. Should I move `img2tex` folder in my project folder? (`flask` projcet)

**Comment by tranngocduvnvp** [comment]
Yes. You can move the img2tex folder to the flask project folder. When running, you just need to stand in the flask folder to run the imjg2tex repo according to the instructions

**Comment by hovkaren** [comment]
Thanks @tranngocduvnvp. Can you please provide example code what I can use this package?

As i understand for input this image 

![math](https://github.com/tranngocduvnvp/Img2LaTex/assets/18136707/270cfb77-7ea9-4816-8ac2-2a22cbfe1ea1)

output should be like this ?

`({a}+{b})^{2}={a}^{2}+2{a}{b}+{b}^{2}
`
